src/lib/cssConfigTemplate.ts

```diff
@@ -1,11 +1,12 @@
-declare var define;
+declare var define: {(arg0: never[], arg1: () => {cssConfig: string}): void; amd: any};
```

Comment:
WebStorm's `infer type from usage` used here. Actually this typing is not vital
![image](https://user-images.githubusercontent.com/439289/110460964-8acdd000-80d7-11eb-8a6a-fa4f234bff6c.png)